DESCRIPTION

```diff
@@ -20,7 +20,6 @@ BugReports: https://github.com/r-lib/usethis/issues
 Depends: 
     R (>= 3.1)
 Imports: 
-    backports (>= 1.1.0),
     clipr (>= 0.3.0),
     clisymbols,
     crayon,
```

Comment:
Should we need to require a specific version of fs?

Reply:
Yes, I should require `fs (>= 1.2.0)` to ensure everything we assume and say about home directory handling on Windows is true.